content/docs/ui/account-and-settings/upgrading-your-account.md

```diff
@@ -15,43 +15,41 @@ Whether you’re a startup or a large enterprise, SendGrid has several plans tai
 
 ## Upgrade your SendGrid plan
 
-### Choose a plan
+Before you can fully use your upgraded account, you need to complete some verification steps for security purposes.
```

Comment:
Before you can upgrade your account, you need to complete some verification steps for security purposes.